rescript/plugin_setup.py

```diff
@@ -1151,10 +1152,14 @@
                                    '\'reference-segment-sequences\' must have '
                                    'in order to extract matching segments '
                                    'from \'input-sequences\'.',
-                'min_seq_len': 'Minimum length of sequence allowed '
-                               'for searching. Any sequence less than '
+                'min_seq_len': 'Minimum length of reference sequence segment '
+                               'allowed for searching. Any sequence less than '
                                'this will be discarded. If not set, default '
-                               'program settings will be used.'},
+                               'program settings will be used.',
+                'max_seq_len': 'Maximum length of reference sequence segment '
+                               'allowed for searching. Any sequence greater '
+                               'than this will be discarded. If not set, '
+                               'default program settings will be used.'},
```

Comment:
```suggestion
                               'than this will be discarded.'},
```
This could be removed with the change from above.